lib/config/wrapper.go

```diff
@@ -271,7 +265,7 @@ func (w *Wrapper) Folders() map[string]FolderConfiguration {
 func (w *Wrapper) FolderList() []FolderConfiguration {
 	w.mut.Lock()
 	defer w.mut.Unlock()
-	return append([]FolderConfiguration(nil), w.cfg.Folders...)
+	return append([]FolderConfiguration(nil), w.cfg.Copy().Folders...)
```

Comment:
Appending to a nil slice already ensures we return a copy.

Reply:
It does not ensure the slices within the folder structs are copied correctly.

Reply:
Oh right, good one. My money is on this as error source :)